tsconfig.json

```diff
@@ -1,12 +1,17 @@
 {
     "compilerOptions": {
-        "module": "amd",
+        "module": "umd",
```

Comment:
why this change? Since VSTS uses AMD I'd like to keep the module format consistent